include/geode/basic/console_progress_logger_client.h

```diff
@@ -35,15 +35,17 @@ namespace geode
         : public ProgressLoggerClient
     {
     public:
-        ConsoleProgressLoggerClient( const std::string& message );
+        ConsoleProgressLoggerClient();
         ~ConsoleProgressLoggerClient();
 
     private:
-        void start( index_t nb_steps ) override;
+        void start( const std::string& message, index_t nb_steps ) override;
 
         void update( index_t current_step, index_t nb_steps ) override;
 
-        void end( index_t current_step, index_t nb_steps ) override;
+        void complete() override;
+
+        void failed() override;
```

Comment:
What are the purpose of these two methods? Is it intentional that one use "ed" but not the other one?

Reply:
The goal is to signal the end of the progress and how it ended.

Reply:
There is probably a better choice of words 😜

Reply:
Ok i would say `completed()` and `failed()`

Reply:
Could you do the changes please?